src/services/action/utils/export.test.ts

```diff
@@ -93,14 +94,14 @@ describe('exportActionsInArchive', () => {
 
     // call on success callback
     expect(result).toBeTruthy();
-    // create files for all views, items, members and memberships, chat messages
-    expect(writeFileSyncMock).toHaveBeenCalledTimes(views.length + 5);
+    // create files for all views, items, members and memberships, chat messages, apps
+    expect(writeFileSyncMock).toHaveBeenCalledTimes(views.length + 6);
```

Comment:
This offset seems a bit magic, I am not sure how we could make it more readable and adaptable to the logic behind it ?

Reply:
idk 🤷‍♀️